src/views/product-integrations-landing/components/searchable-integrations-list/index.tsx

```diff
@@ -345,7 +345,18 @@ export default function SearchableIntegrationsList({
 
 			<div className={classNames(s.results, s.mobile_only)}>{resultText}</div>
 
-			<PaginatedIntegrationsList integrations={filteredIntegrations} />
+			<PaginatedIntegrationsList
+				integrations={filteredIntegrations}
+				onClearFiltersClicked={() => {
+					setOfficialChecked(false)
+					setPartnerChecked(false)
+					setCommunityChecked(false)
+					setComponentCheckedArray(componentCheckedArray.map((v, i) => false))
+					setFlagsCheckedArray(flagsCheckedArray.map((v, i) => false))
+					setFilterQuery('')
+					resetPage()
+				}}
```

Comment:
💭  Just an idea:

Maybe we can update the existing `handleClearFilters` function to include `setFilterQuery('')` and pass that directly to `onClearFiltersClicked={handleClearFilters}`?